### PR TITLE
Fixes #15274: Handle errors in log filter

### PR DESCRIPTION
**Pull request by amousset**
https://issues.rudder.io/issues/15274

### CODE REVIEW

**Comment by amousset** [comment]
PR updated with a new commit

File: relay/sources/Makefile [hunk 1]
```diff
@@ -149,7 +149,7 @@ lint:
 
 check: lint
 	cd relayd && cargo test
-	cd relayd && cargo audit
+	cd relayd && cargo audit --ignore RUSTSEC-2019-0011 #https://github.com/crossbeam-rs/crossbeam/issues/401
```
Comment (peckpeck) [inline relay/sources/Makefile]: What is this ?

Comment (amousset) [reply relay/sources/Makefile]: To temporarily ignore the security alert and make the tests pass, I'll open an issue to remove it once fixed upstream.

**Comment by amousset** [comment]
OK, squash merging this PR

**Pull request merged**